ui/server/src/test/scala/pl/touk/nussknacker/ui/process/migrate/ProcessModelMigratorSpec.scala

```diff
@@ -34,6 +34,16 @@ class ProcessModelMigratorSpec extends FlatSpec with BeforeAndAfterEach with Pat
     processor shouldBe ServiceRef(ProcessTestData.otherExistingServiceId, List())
   }
 
+  it should "migration should not return empty migration result" in {
+
+    val migrationResultOpt: Option[MigrationResult] = migrateByVersionsOpt(Some(1), 1)
+
+    migrationResultOpt.map(o => println(o.toUpdateAction(ProcessId(1L))))
```

Comment:
Please don't use system out. It will be hard to reduce numer of such logs by simple changes in logback configuration. Can it be printed using logger?

Reply:
Hah, it should be removed - I missed it while cleaning after debugging. Deleted.